python/plugins/processing/algs/qgis/TilesXYZ.py

```diff
@@ -565,6 +578,21 @@ def initAlgorithm(self, config=None):
                                                                 self.tr('Output html (Leaflet)'),
                                                                 self.tr('HTML files (*.html)'),
                                                                 optional=True))
+        html_title_param = QgsProcessingParameterString(self.HTML_TITLE,
+                                                        self.tr('Title (Leaflet html output)'),
```

Comment:
```suggestion
                                                        self.tr('Title (Leaflet HTML output)'),
```

although I'd lean toward something along the lines of "Leaflet HTML output title" / "Leaflet HTML output attribution" / "Include OpenStreetMap basemap in Leaflet HTML output"

Reply:
Thanks, this sounds much better indeed.

adressed per https://github.com/qgis/QGIS/pull/51522/commits/257fafa006557595e581c43c88a0b9b0dbe8c691